### DISCUSSION TITLE
Verification message cannot be submitted

### DISCUSSION THREAD

**Discussion by JasonZhaoWW** [community/community#133758] [GitHub Education]
### Hi everyone,

After clicking "Process my application", it redirect to "Upload proof" page and clear the filled information.

**Comment by SohanKafle** [comment]
Hi [JasonZhaoWW](https://github.com/JasonZhaoWW),

If your application information clears when trying to upload proof, try these steps:

**1. Clear Cache:** Clear your browser cache or try using an incognito window.
**2. Different Browser:** Use a different browser to submit your application.
**3. Contact Support:** If the issue persists, contact [GitHub Education Support](https://support.github.com/contact/education) for assistance.

**Reply by JasonZhaoWW** [reply]
Sorry it doesn't work, the result is the same😭

**Reply by SohanKafle** [reply]
Sorry to hear that. Contact [GitHub Education Support](https://support.github.com/contact/education) for assistance.
